src/metax_api/api/rpc/base/views/file_rpc.py

```diff
@@ -60,4 +71,4 @@ def flush_project(self, request): # pragma: no cover
 
         _logger.info('Permanently deleted all files and directories from project %s' % project)
 
-        return Response(data=None, status=status.HTTP_204_NO_CONTENT)
+        return Response(data=None, status=status.HTTP_204_NO_CONTENT)
```

Comment:
Newline at end of files are fine! Discussions https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file